src/io/flutter/run/daemon/DevToolsService.java

```diff
@@ -240,6 +242,7 @@ private void logExceptionAndComplete(String message) {
 
   private void logExceptionAndComplete(Exception exception) {
     LOG.error(exception);
+    FlutterInitializer.getAnalytics().sendException(StringUtil.getThrowableText(exception), false);
```

Comment:
We need to first audit these exceptions as part of this cl and change the name so it is clear we are logging the exceptions to analytics. Currently they are including the project name and a file system path both of which should not be included in exceptions that are always logged to analytics.

Reply:
I don't think the project name and system path will be logged to analytics - this line makes the exception into text (which only has names of files like `io.flutter.run.daemon.DevToolsService`) and then goes through the `sendPayload` analytics function, which appends info like plugin version. Am I missing analytics getting sent through another path?

Reply:
(example of what would go into analytics description field)
```
java.lang.Exception: test exception
	at io.flutter.run.daemon.DevToolsService.logExceptionAndComplete(DevToolsService.java:241)
	at io.flutter.run.daemon.DevToolsService.access$500(DevToolsService.java:46)
	at io.flutter.run.daemon.DevToolsService$2.onTextAvailable(DevToolsService.java:188)
	at jdk.internal.reflect.GeneratedMethodAccessor61.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Metho
```

Reply:
The calls that concern me slightly are 
        logExceptionAndComplete("Unable to find daemon command for project: " + project);
and 
logExceptionAndComplete("Handler was null for command: " + command.toString());
that could expose a file path on their machine.

Reply:
Ohhh I see, I'll check through these.

Reply:
I just removed the contextual information that could be private. It might be nice to have this information locally logged if users report bugs, but I can always add that back later.